src/EditorFeatures/Core.Wpf/InlineRename/NullTextBufferException.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.InlineRename
+{
+    // Used to aid the investigation of https://github.com/dotnet/roslyn/issues/7364
+    internal class NullTextBufferException : Exception
```

Comment:
Nit: I'd move this as private class right beside InitializeOpenBuffers so somebody doesn't get the idea of depending on it.